### ISSUE TITLE
Function to reference the graphql(name) defined on an enum

### ISSUE THREAD

**Issue by burkematthew** [async-graphql/async-graphql#1518]
Hi,

I have an enum defined as such

```
#[derive(Enum, Copy, Clone, Eq, PartialEq, Serialize, Debug, ToSql, FromSql)]
#[postgres(name = "home_quality_check_status")]
pub enum HomeQualityCheckStatus {
  #[postgres(name = "Ready")]
  Ready,
  #[graphql(name = "Not Ready")]
  #[postgres(name = "Not Ready")]
  NotReady,
}
```

If I have a field defined as:

```
pub home_quality_check_status: HomeQualityCheckStatus
```

is there a way to pull either the value of `Ready` or `Not Ready` as defined by the `#[graphql(name =` attribute?  If not, I'd like to request this as a feature enhancement!

**Comment by sunli829** [comment]
You can implement the `Display trait` for `HomeQualityCheckStatus`

```rust
impl Display for HomeQualityCheckStatus {
    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
        match self {
            HomeQualityCheckStatus::Ready => f.write_str("Ready"),
            ...
        }
    }
}
```

**Comment by ziimakc** [comment]
Same question, I've already have display trait and would want to reuse it as instead duplicating `#[graphql(name = "BAD_REQUEST")]`

**Comment by sunli829** [comment]
released in `v7.0.5`
